specification/src/main/asciidoc/platform/BrandMapping.adoc

```diff
@@ -0,0 +1,114 @@
+[cols=2, options=header]
+.Java EE to Jakarta EE Mapping
+|===
+|Java EE Technology
+|Jakarta EE Technology
+
+|Java™ Platform, Enterprise Edition 8
+|Jakarta™ EE Platform 8
+
+|Enterprise JavaBeans™ 3.2
+|Jakarta™ Enterprise Beans 3.2
+
+|JavaServer Pages™ 2.3
+|Jakarta™ Server Pages 2.3
+
+|Expression Language 3.0
+|Jakarta™ Expression Language 3.0
+
+|Common Annotations for the Java Platform 1.3
+|Jakarta™ Annotations 1.3
+
+|Java™ Servlet 4.0
+|Jakarta™ Servlet 4.0
+
+|Java™ Message Service 2.0
+|Jakarta™ Messaging 2.0
+
+|Java™ Transaction API 1.2
```

Comment:
Should be 1.3.

Reply:
Wasn't it JTA 1.2 in Java EE 8?  We allowed the change to separate the Java EE vs Java SE interfaces after Java EE 8 was released, right?  So, Java EE 8 was still with JTA 1.2.  JTA 1.3 is post Java EE 8, but part of Jakarta EE 8.

Reply:
Added a section that explicitly explains the creation of 1.3 and that JTA 1.2, JTA.1.3 and Jakarta Transaction 1.3 are functionally equivalent.